### ISSUE TITLE
[AdaptiveCards- Designer - Target Version]: At 200%, Target version button is getting cropped.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8172]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: [https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)
OS Version: 22H2(OS Build 25247.1000)
Browser Version: Version 110.0.1543.0 (Official build) dev (64-bit)

**Pre-requisites:**
1. Set the screen resolution to 1280x768.
2. Set text size to 200% (Settings -> Ease of access -> Make text bigger). No custom scaling required, as scaling will be at default 100%.

**Repro Steps:**
1. Open above URL with valid credential.
2. Navigate to 'Designer' link and press enter key to activate it.
3. Navigate to 'Redo' button and press tab key.
4. Navigate to 'Target Version' button and observe the issue.

**Actual Result:**
At 200%, Target version button is getting cropped and does not display properly.

**Expected Result:**
Target version button should be displayed properly at 200% and it should not be cropped.

**User Experience:**
It will impact the low vision user if the any element on is getting cropped at 200%.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B2320371F-0114-4395-93C0-EDF6443F0F20%7D&file=MAS%201.4.4%20%E2%80%93%20Resize%20Text.docx&action=default&mobileredirect=true

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

<img width="1914" alt="MAS1 4 4-At 200%, Target Version button is getting cropped" src="https://user-images.githubusercontent.com/95913748/208670765-85e4ca42-f694-4658-b9ee-07e073ade897.png">

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is not repro, hence closing this issue.

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/a71b37d2-c80c-4fa7-8ca0-7eb8f08aa8e2)
